<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Udp client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="5">
    <widget class="QPushButton" name="buttonSendMessage">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="buttonConnect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelSendToInfo">
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="buttonDisconnect">
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="Line" name="separatorLog">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QPlainTextEdit" name="editLog"/>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QPlainTextEdit" name="editMessage"/>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QComboBox" name="comboAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QLineEdit" name="editPort">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>10001</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="labelInfo">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboSendToAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="editSendToPort">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>10000</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboAddress</tabstop>
  <tabstop>editPort</tabstop>
  <tabstop>buttonConnect</tabstop>
  <tabstop>buttonDisconnect</tabstop>
  <tabstop>comboSendToAddress</tabstop>
  <tabstop>editSendToPort</tabstop>
  <tabstop>editMessage</tabstop>
  <tabstop>buttonSendMessage</tabstop>
  <tabstop>editLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
